packages/meta-harness/tests/test_scenario_docker.py

```
@@ -1373,54 +1373,92 @@ def test_broker_env_allows_evaluate_and_judge_when_pinned_to_the_same_model() ->
+    review)."""
     config = copy.deepcopy(mh.DEFAULTS)
-    commands = []
+    config["storage"]["root"] = "relative/not-absolute"
 
-    def runner(command, **_kwargs):
-        commands.append(command)
-        return _completed(stdout="sha256:" + "3" * 64)
+    def runner(*_args, **_kwargs):
+        raise AssertionError("ensure_images must fail before touching Docker")
+
+    with pytest.raises(docker.dcli.DockerCliError, match="storage.root must be an absolute path"):
```

Ruff (0.15.21)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)